helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -224,6 +224,69 @@ spec:
     - expr: sum by (service, area) (raw_slo_errors{area="managed-apps"} - raw_slo_errors{area="managed-apps"}) + 1-0.99
       record: slo_target
 
+    # core k8s components internal API requests
+    # record number of requests.
+    - expr: sum(rest_client_requests_total{app=~"kube-controller-manager|kube-scheduler"}) by (cluster_id,app)
```

Comment:
I thought we were unimpressed by the useful of this metric?

Reply:
But this should catch errors with the admission controllers not being running for example.
I wish I could test this easily without impacting oncallers.